modules/gapi/perf/gpu/gapi_imgproc_perf_tests_gpu.cpp

```diff
@@ -128,20 +128,20 @@ INSTANTIATE_TEST_CASE_P(SobelPerfTestGPU32F, SobelPerfTest,
 
 INSTANTIATE_TEST_CASE_P(LaplacianPerfTestGPU, LaplacianPerfTest,
                         Combine(Values(ToleranceFilter(1e-4f, 0.01).to_compare_f()),
-                                Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1),
-                                Values(3, 5),
+                                Values(CV_32FC1),
```

Comment:
I don't think it's good if types checked on CPU differ from ones on GPU

Reply:
Fixed.